Doc/library/contextlib.rst

```diff
@@ -80,6 +80,35 @@ Functions and classes provided:
       Use of :class:`ContextDecorator`.
 
 
+.. decorator:: asynccontextmanager
+
+   Similar to :func:`~contextlib.contextmanager`, but works with
+   :term:`coroutines <coroutine>`.
```

Comment:
"works with coroutine" isn't really descriptive. I'd rephrase to something akin to "but creates asynchronous context managers".